Util/model_parser_cxx/model_parser_data.H

```diff
@@ -26,6 +34,17 @@ namespace model
         amrex::Array1D<amrex::Real, 0, NPTS_MODEL-1> r;
     };
 
+    // Tolerance used for getting the total star mass equal to the desired mass.
+    // It can be reasonably small, since there will always be a central density
+    // value that can give the desired mass on the grid.
+
+    const amrex::Real mass_tol = 1.0e-6_rt;
```

Comment:
I never changed these in practice, so I just hardcoded them here. But we could make these runtime parameters later if we need to.